src/components/header/search-form/SearchForm.tsx

```diff
@@ -0,0 +1,47 @@
+import './SearchForm.css';
+import {searchFunction} from "../../../models/search";
+import {useState} from "react";
+
+function SearchForm({ searchChange }: {searchChange: searchFunction}) {
+    const [formData, setFormData] = useState({
+        query: '',
+    });
```

Comment:
Not only a callback but the Initial search query should be passed to the component as a property too.

Reply:
Added, thank you